guard/src/commands/validate.rs

```diff
@@ -302,7 +302,11 @@ or rules files.
                     for file in walk_dir(base, cmp) {
                         if file.path().is_file() {
                             let name = file
-                                .file_name()
+                                .path()
+                                // path output occasionally includes double slashes '//'
+                                // without calling canonicalize()
+                                .canonicalize()
+                                .unwrap()
```

Comment:
Is there a specific reason we chose to call unwrap() instead of handling the error?

Reply:
Same as below - went with this since it's only used in test but can handle differently with a message if you think thats better

Reply:
Hmm, I believe this to be incorrect. This gets called anytime the command is invoked with a data argument